alternator/executor.cc

```diff
@@ -4792,7 +4793,7 @@ static lw_shared_ptr<keyspace_metadata> create_keyspace_metadata(std::string_vie
     // used by default on new Alternator tables. Change this initialization
     // to 0 enable tablets by default, with automatic number of tablets.
     std::optional<unsigned> initial_tablets;
-    if (sp.get_db().local().get_config().enable_tablets()) {
+    if (feat.tablets) {
```

Comment:
Does this mean that alternator keyspaces will be using tablets unconditionally from now on?